mne/viz/_brain/_brain.py

```diff
@@ -2611,15 +2643,18 @@ def screenshot(self, mode='rgb', time_viewer=False):
             # renderer tostring_rgb() size. So let's directly use what
             # matplotlib does in lib/matplotlib/backends/backend_agg.py
             # before calling tobytes():
-            trace_img = np.asarray(
-                canvas.renderer._renderer).take([0, 1, 2], axis=2)
+            #
+            # trace_img = np.asarray(
+            #     canvas.renderer._renderer).take([0, 1, 2], axis=2)
+            #
```

Comment:
This will be in `git` history if we ever need it (but it used private attributes anyway so hopefully we don't)